pkg/plugins/policies/core/matchers/dataplane_test.go

```diff
@@ -206,4 +205,109 @@ var _ = Describe("MatchedPolicies", func() {
+			Entry("k8s zone", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneK8s,
+				policyMeta:    test_resources.ZoneK8s,
+				goldenFile:    buildGoldenFilePath("k8s-zone", givenResources.testName),
+			}),
+			Entry("policy global uni, dpp uni", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneUni,
+				policyMeta:    test_resources.SystemPolicy(test_resources.GlobalUni),
+				goldenFile:    buildGoldenFilePath("policy-from-global-uni-zone-uni", givenResources.testName),
+			}),
+			Entry("policy global uni, dpp k8s", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneK8s,
+				policyMeta:    test_resources.SystemPolicy(test_resources.GlobalUni),
+				goldenFile:    buildGoldenFilePath("policy-from-global-uni-zone-k8s", givenResources.testName),
+			}),
+			Entry("policy global k8s, dpp uni", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneUni,
+				policyMeta:    test_resources.SystemPolicy(test_resources.GlobalK8s),
+				goldenFile:    buildGoldenFilePath("policy-from-global-k8s-zone-uni", givenResources.testName),
+			}),
+			Entry("policy global k8s, dpp k8s", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneK8s,
+				policyMeta:    test_resources.SystemPolicy(test_resources.GlobalK8s),
+				goldenFile:    buildGoldenFilePath("policy-from-global-k8s-zone-k8s", givenResources.testName),
+			}),
+			Entry("policy global k8s, dpp uni", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneUni,
+				policyMeta:    test_resources.SystemPolicy(test_resources.GlobalUni),
+				goldenFile:    buildGoldenFilePath("policy-global-uni-dpp-k8s", givenResources.testName),
+			}),
+			Entry("policy synced from other k8s zone", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneUni,
+				policyMeta:    test_resources.ProducerPolicy(test_resources.SyncToUni(test_resources.ZoneK8s)),
+				goldenFile:    buildGoldenFilePath("policy-form-k8s-to-uni", givenResources.testName),
+			}),
+			Entry("policy synced from other uni zone", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneK8s,
+				policyMeta:    test_resources.ProducerPolicy(test_resources.SyncToK8s(test_resources.ZoneUni)),
+				goldenFile:    buildGoldenFilePath("policy-form-uni-to-k8s", givenResources.testName),
+			}),
+			Entry("policy synced from other uni zone to uni", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneUni,
+				policyMeta:    test_resources.ProducerPolicy(test_resources.SyncToUni(test_resources.ZoneUni)),
+				goldenFile:    buildGoldenFilePath("policy-form-uni-to-uni", givenResources.testName),
+			}),
+			Entry("policy synced from other k8s zone to k8s", dataplaneTestCase{
+				dataplaneMeta: test_resources.ZoneK8s,
+				policyMeta:    test_resources.ProducerPolicy(test_resources.SyncToK8s(test_resources.ZoneK8s)),
+				goldenFile:    buildGoldenFilePath("policy-form-k8s-to-k8s", givenResources.testName),
+			}),
+		)
+	}, generateTableEntries(filepath.Join("testdata", "matchedpolicies", "dataplane-kind")))
```

Comment:
btw should be pretty easy to replace it with `test.EntriesForFolder("matchedpolicies/dataplane-kind")`

Reply:
But it only reads one file with `.input` suffix and I want to keep dataplane and policies in separate files to avoid complexity of splitting them in test later on